spec/system/home_page_spec.rb

```diff
@@ -44,7 +44,7 @@ def i_can_see_the_placements_service_name_in_the_header
   end
 
   def and_i_can_see_the_claims_service_value_in_the_page
-    expect(page).to have_css(".govuk-summary-list__value", text: "Claims")
+    expect(page).to have_css(".app-primary-navigation__link", text: "Claims")
   end
 
   def and_i_can_see_the_placements_service_value_in_the_page
```

Comment:
This may mean we don't need this spec anymore, right? As there is no longer a "home page".